rundeckapp/grails-app/views/scheduledExecution/_optEdit.gsp

```diff
@@ -388,7 +388,7 @@
                         </div>
                 <feature:enabled name="optionValuesPlugin">
                     <!--List OptionValuesPlugins here -->
-                    <g:each in="${optionValuesPlugins}" var="optionValPlugin">
+                    <g:each in="${optionValuesPlugins?.sort{a,b->a.key<=>b.key}}" var="optionValPlugin">
```

Comment:
just curious @imad6639 
maybe this sort can be done in the controller/service instead of the view (it could be better to test it)